### DISCUSSION TITLE
TS100 Thermal Runaway while soldering

### DISCUSSION THREAD

**Discussion by Tekz42** [Ralim/IronOS#1944] [Q&A] [answered]
Hey Guys, My TS100 keeps going into Thermal Runaway while soldering small connections with small copper. Im using V2.22 and its making it very hard to solder.

**Comment by Ralim** [comment]
Hello,
What power supply are you using? The most common cause of this tripping is when the tip cant maintain temperature; which often means your using an insufficient power supply.

(Could also be tuning, but want to rule out PSU first)

**Reply by Tekz42** [reply]
12v 2A AC adapter or a 2200mAh 3s LiPo 25-50C discharge

**Reply by Ralim** [reply, marked as answer]
Ah okay that will be why you are running into this.
The TS100 tip is ~ 8 ohms; so at 12V you are only getting around 18W to the tip.
The iron works exponentially batter at higher voltages.

**Reply by Tekz42** [reply]
Oh Righto, so this will be a problem no matter what with a 3s LiPo?

**Reply by Ralim** [reply]
Unfortunately yeah. 
I can back off the threshold at which it occurs, but what its really saying is "I put in max power for a while and temperature never came up". So this trips as a safety to prevent runaway when tips fail (and then get very, very,very hot).

Even without the warning you will find soldering in this situation very frustrating as it will take a long time for stuff to heat up and melt.

I would strongly suggest aiming for a 18-24V supply if at all possible.

**Reply by Tekz42** [reply]
Thats a bummer. This is my field Iron as it is mainly used for fpv and what I have are max 4s LiPo. Never had this issue with stock firmware but much prefer this for its functionality. Id rather have the Iron hold that temp rather than it going into cool down and losing the joint. Maybe that be a feature when using a LiPo??

**Reply by Ralim** [reply]
Definitely can be re-tuned or an option added to turn it off.
Can you make an issue so that its tracked^ if you get time ?

**Reply by Tekz42** [reply]
Issue has been added #1946
Cheers

**Reply by Ralim** [reply]
Thank you. I'll move any further work on this over there for tracking.
